providers/azure/README.md

```diff
@@ -197,35 +197,31 @@ Run these commands to deploy the virtual kubelet which connects your Kubernetes
 If your cluster is an AKS cluster:
 
 ```cli
-export VK_RELEASE=virtual-kubelet-for-aks-0.1.3
+export VK_RELEASE=virtual-kubelet-for-aks-0.1.6
 ````
 
 For any other type of Kubernetes cluster:
 
 ```cli
-export VK_RELEASE=virtual-kubelet-0.1.1
+export VK_RELEASE=virtual-kubelet-0.1.3
```

Comment:
since 0.2.0 is merged, shall we update to it?
notice that the values structure is changed. we need to update accordingly.

Reply:
Yep! I was actually updating this PR to do that, hadn't pushed a new commit yet :)